### DISCUSSION TITLE
Hides the cursor after pressing a key in the new version of the editor

### DISCUSSION THREAD

**Discussion by ghost** [codesandbox/codesandbox-client#8373] [General]
In the new version of editor after key pressed after a half of seconds cursor hides. How to fix it in the settings? Or switch editor to older version?

https://github.com/codesandbox/codesandbox-client/assets/50989834/5660a828-48a0-48c4-918a-7dd5b28ce7d0

**Comment by JamesACS** [comment]
Hello @Dvisik 

It looks like you may have VIM enabled. 

If you disable this in your environments settings - it should resolve the problem
